pkgs/dartpad_ui/lib/html_view/html_view.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2025, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import '_stub.dart' if (dart.library.js_util) '_web.dart';
```

Comment:
`dart:js_util` is deprecated, so consider updating this and other conditional imports to `dart:js_interop`:

```suggestion
import '_stub.dart' if (dart.library.js_interop) '_web.dart';
```

Reference: https://dart.dev/interop/js-interop/package-web#conditional-imports